app/src/main/java/com/markkolenbrander/capstonenewsapp/models/Article.kt

```diff
@@ -1,5 +1,6 @@
 package com.markkolenbrander.capstonenewsapp.models
 
+
 data class Article(
```

Comment:
Tomorrow I'll go over this a bit more, but great work on attempting to make Article and Source `Parcelable` types! Note for the future you can use the `@Parcelize` annotations (have to add the `'kotlin-parcelize' plugin id at the top of your build.gradle file) in order to have these types be set as `Parcelable` with no boiler-plate code:

```suggestion
@Parcelize
data class Article(
    ...
) : Parcelable
```